src/minimega/vm.go

```diff
@@ -78,6 +79,11 @@ type VM interface {
 	// NetworkDisconnect updates the VM's config to reflect that the specified
 	// tap has been disconnected.
 	NetworkDisconnect(int) error
+
+	// Qos functions
+	GetQos() [][]string
```

Comment:
Not sure how we would include the vm's network information (bridge name, tap name, etc) if this function returned []bridge.QosParams.